<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGISRedLayerManagementDialog</class>
 <widget class="QDialog" name="QGISRedLayerManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGISRed: Layer Management</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/plugins/QGISRed/images/iconLayerManagement.png</normaloff>:/plugins/QGISRed/images/iconLayerManagement.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>CRS:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbCRS">
         <property name="text">
          <string>WGS 84</string>
         </property>
         <property name="cursorPosition">
          <number>6</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::VisualMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btSelectCRS">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="gbElements">
       <property name="title">
        <string>Basic Elements Visibility in Legend</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="cbPipes">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Pipes</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btPipes">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Pipes Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
                <property name="autoDefault">
                 <bool>true</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QCheckBox" name="cbJunctions">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Junctions</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btJunctions">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Junction Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QCheckBox" name="cbTanks">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Tanks</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btTanks">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Tanks Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QCheckBox" name="cbReservoirs">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Reservoirs</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btReservoirs">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Reservoirs Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QCheckBox" name="cbValves">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Valves</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btValves">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Valves Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QCheckBox" name="cbPumps">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Pumps</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btPumps">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Pumps Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QCheckBox" name="cbDemands">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Demands</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btDemands">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Demands Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QCheckBox" name="cbSources">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Sources</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btSources">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Sources Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>22</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Digital Twin Visibility in Legend</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QCheckBox" name="cbConnections">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Service Connections</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btConnections">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Service Connections Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_18">
              <item>
               <widget class="QCheckBox" name="cbIsolatedValves">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Isolation Valves</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btIsolatedValves">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Isolation Valves Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <widget class="QCheckBox" name="cbHydrants">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Hydrants</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btHydrants">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Hydrants Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_20">
              <item>
               <widget class="QCheckBox" name="cbPurgeValves">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Washout Valves</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btPurgeValves">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Whasout Valves Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QCheckBox" name="cbAirReleases">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>AirRelease Valves</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btAirReleases">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create AirRelease Valves Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QCheckBox" name="cbMeters">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Meters</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btMeters">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Create Manometers Layer</string>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
                <property name="autoDefault">
                 <bool>true</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btAccept">
         <property name="text">
          <string>Accept</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
